src/components/profile/AvatarCircle.js

```diff
@@ -71,37 +74,62 @@ export default function AvatarCircle({
     [resolvedColor, colors, isDarkMode]
   );
 
+  const items = useMemo(
+    () => ({
+      menuItems: menuOptions?.map(label => ({
+        actionKey: label,
+        actionTitle: label,
+      })),
+    }),
+    [menuOptions]
+  );
+
+  const handlePressMenuItem = useLatestCallback(
+    e => {
+      const index = items.menuItems?.findIndex(
+        item => item.actionKey === e.nativeEvent.actionKey
+      );
+      onSelectionCallback(index);
+    },
+    [onSelectionCallback, items.menuItems]
+  );
+
+  const Wrapper =
+    ios || !isAvatarPickerAvailable ? React.Fragment : ContextMenuButton;
+
   return (
-    <ButtonPressAnimation
-      disabled={!isAvatarPickerAvailable}
-      enableHapticFeedback={isAvatarPickerAvailable}
-      marginTop={2}
-      onPress={onPress}
-      overflowMargin={30}
-      pressOutDuration={200}
-      scaleTo={isAvatarPickerAvailable ? 0.9 : 1}
-      {...props}
-    >
-      <ShadowStack
-        {...position.sizeAsObject(AvatarCircleSize)}
-        backgroundColor={overlayStyles ? 'rgb(51, 54, 59)' : colors.white}
-        borderRadius={AvatarCircleSize}
-        marginBottom={12}
-        shadows={shadows[overlayStyles ? 'overlay' : 'default']}
-        {...(android && {
-          height: 64,
-          width: 64,
-        })}
+    <Wrapper handlePressMenuItem={handlePressMenuItem} menuConfig={items}>
+      <ButtonPressAnimation
+        disabled={!isAvatarPickerAvailable}
+        enableHapticFeedback={isAvatarPickerAvailable}
+        marginTop={2}
+        {...(ios && { onPress })}
+        overflowMargin={30}
+        pressOutDuration={200}
+        scaleTo={isAvatarPickerAvailable ? 0.9 : 1}
+        {...props}
       >
-        {image ? (
-          <ImageAvatar image={image} size="large" />
-        ) : (
-          <AvatarCircleView backgroundColor={resolvedColor}>
-            <FirstLetter>{accountSymbol}</FirstLetter>
-            {!overlayStyles && <InnerBorder opacity={0.02} radius={65} />}
-          </AvatarCircleView>
-        )}
-      </ShadowStack>
-    </ButtonPressAnimation>
+        <ShadowStack
+          {...position.sizeAsObject(AvatarCircleSize)}
+          backgroundColor={overlayStyles ? 'rgb(51, 54, 59)' : colors.white}
```

Comment:
We use that color in two places, maybe it makes sense to move it to colors

Reply:
they ar indeed used in two places, but i don't know what connects those two usages. Probably this is accidental. Do you have any idea how to connect them?